README.rst

```diff
@@ -101,15 +101,14 @@ You can update ``SIGNALFX_ENDPOINT_URL`` where both metrics and traces will be s
 
 **Option 2**
 
-You can update ``SIGNALFX_ENDPOINT_URL`` and ``SIGNALFX_METRICS_URL`` where traces will be sent to the gateway and metrics will go through POPS, respectively. 
+You can update ``SIGNALFX_ENDPOINT_URL`` and ``SIGNALFX_METRICS_URL`` where traces will be sent to the gateway and metrics will go directly, respectively. 
```

Comment:
"directly to SignalFx", "directly to SignalFx ingest"? just "directly" without the object sounds weird ;)

Reply:
Fixed